test/exe/Main.hs

```diff
@@ -1132,11 +1132,11 @@ findDefinitionAndHoverTests = let
   , test yes    yes    mclL37 mcl    "top-level fn 2nd clause         #246"
   , test yes    yes    spaceL37 space "top-level fn on space #315"
   , test no     broken docL41 doc    "documentation                     #7"
-  , test no     broken eitL40 kindE  "kind of Either                  #273"
-  , test no     broken intL40 kindI  "kind of Int                     #273"
+  , test no     yes    eitL40 kindE  "kind of Either                  #273"
```

Comment:
I’m struggling to figure out the part of this PR that adds kind information, could you help me out here?

Reply:
It starts here https://github.com/digital-asset/ghcide/pull/317/files#diff-9e4907094a15701d642e91bb4b7d5bb7R181